### DISCUSSION TITLE
[DONE] Team Dashboard: list disabled / failed checks

### DISCUSSION THREAD

**Discussion by jone** [ParetoSecurity/pareto-mac#67] [Ideas]
We've figured out that we'd like to see in the team dashboard, which specific checks are disabled or failing on a machine.
We'd like to see that, because we think that not all checks are equally bad when disabled or failing and this would allow us to act on a case-to-case basis.

**Comment by dmurko** [comment]
We had internal discussions about this if it's too invasive so your feedback helps a lot. Thanks!

**Comment by zupo** [comment]
I still personally think it's too invasive. At least for my machine, I would prefer just the number, not an exact list of my mistakes.

Maybe we can do it as a configuration option? Or would that make the app too complex? I'll give this more thought.

**Reply by dmurko** [reply]
In some cases, not having an invasive check might cause more invasiveness by someone requesting a screen share and going through the computer with you to fix that issue. By having the check, they can just send you instructions and push you to fix it asap if it's important.

**Reply by zupo** [reply]
Fair point!

**Comment by zupo** [comment]
Done!

You can now click on a device name in the device list to get more information about the device. 
<img width="1339" alt="Screenshot 2021-11-16 at 09 47 17" src="https://user-images.githubusercontent.com/311580/141962106-4efbe1ca-5bc2-4e12-9cdd-5dd612ced655.png">


We still need to add MacOS version information to this view, coming soon.

**Comment by zupo** [comment]
> We still need to add MacOS version information to this view, coming soon.

Also now done.
